conductr-cloudformation.yaml

```diff
@@ -186,6 +197,62 @@ Resources:
       ToPort: 10999
       SourceSecurityGroupId: !Ref NodesSecurityGroup
 
+  # The ConductR Cluster Instances
+  SeedNode0:
+    Type: AWS::EC2::Instance
+    DependsOn: VPCGatewayAttachment
+    Properties:
+      ImageId: ami-dd679fcb # TODO: look up based on region (see "Mappings" above)
+      KeyName: !Ref KeyName
+      InstanceType: "t2.xlarge" # TODO: make configurable?
+      AvailabilityZone: !GetAtt Subnet0.AvailabilityZone
+      NetworkInterfaces:
+        - DeviceIndex: "0"
+          AssociatePublicIpAddress: "true"
+          GroupSet: [ !Ref NodesSecurityGroup ]
+          SubnetId: !Ref Subnet0
+      Tags:
+        - Key: Name
+          Value: Lightbend Cluster Node 0 (Seed)
```

Comment:
Suggest keeping seed to Role tag only. Seed nodes should not be presented as too special.